.github/workflows/test_qlib_from_pip.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
```

Comment:
Suggest changing this name. Otherwise they will all be "Test" and undistinguishable.